unidist/core/backends/mpi/core/worker/request_store.py

```diff
@@ -190,6 +190,10 @@ def process_get_request(self, source_rank, data_id):
             Rank number to send data to.
         data_id: unidist.core.backends.common.data_id.DataID
             `data_id` associated data to request
+        is_blocking_op : bool, default: False
+            # Whether the get request should be blocking or not.
+            # If ``True``, the request should be processed immediatly
+            # even for a worker since it can get into controller mode.
```

Comment:
```suggestion
            Whether the get request should be blocking or not.
            If ``True``, the request should be processed immediatly
            even for a worker since it can get into controller mode.
```